wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -30,6 +30,7 @@ enum CoreDataMessagingMigrationVersion: String, CaseIterable {
     // MARK: -
 
     // Note: add new versions here in first position!
+    case version2_111 = "zmessaging2.111.0"
```

Comment:
We will need to move all the model versions in develop from 2.111 up one (e.g 2.111 -> 2.112, 2.112 -> 2.113) because this 2.111 version is not the same as the 2.111 in develop.

Reply:
I know